tests/regr_test.py

```diff
@@ -315,15 +335,20 @@ def main() -> ReturnCode:
         platforms_to_test, versions_to_test = SUPPORTED_PLATFORMS, SUPPORTED_VERSIONS
     else:
         platforms_to_test = args.platforms_to_test or [sys.platform]
-        versions_to_test = args.versions_to_test or [f"3.{sys.version_info[1]}"]
+        versions_to_test = args.versions_to_test or [PYTHON_VERSION]
 
     code = 0
+
```

Comment:
spurious edit ;)

```suggestion
```

Reply:
actually the line `code = 0` is not needed :)

Reply:
Kill it with fire! It must have been accidentally left in after my big refactor in #10714 :D